### ISSUE TITLE
jsdom.env is not defined

### ISSUE THREAD

**Issue by raphaelsoul** [bda-research/node-crawler#366]
https://github.com/bda-research/node-crawler/blob/e8a51d794c1a712bef65594556311218fdcb9fb7/lib/crawler.js#L157

reproduce code
```
var jsdom = require('jsdom');
var Crawler = require('crawler');

var c = new Crawler({ jQuery: jsdom, jquery: jsdom });

c.queue({
  jQuery: {
    jsdom
  },
  uri: "https://www.baidu.com",
  callback: (err, res, done) => {
    console.log(err, res, done)
  }
})
```

This leads to 
![image](https://user-images.githubusercontent.com/8615787/99024340-5a4ead00-25a1-11eb-988b-e1fda48da66b.png)

By the way the statement about jsdom seems to be outdated. pass  `{ jQuery: require('jsdom') }` not works

**Comment by mike442144** [comment]
You are right, the jsdom option is outdated. I recommend you to adopt  `puppeteer` to get what you want instead.
